glib/src/object.rs

```diff
@@ -3232,12 +3232,13 @@ fn validate_signal_arguments(type_: Type, signal_query: &SignalQuery, args: &mut
     }
 }
 
-impl ObjectClass {
```

Comment:
Can you mark the type as manual and see if any extra-docs gets embedded?

Reply:
That seems very optimistic, but let's give it a try :)

Reply:
CI passed. Ok to merge?

Reply:
let us do that